io/include/detray/io/json/json_io.hpp

```diff
@@ -0,0 +1,13 @@
+/** Detray library, part of the ACTS project (R&D line)
+ *
+ * (c) 2022 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+#include "detray/io/json/json_algebra_io.hpp"
+#include "detray/io/json/json_geometry_io.hpp"
+#include "detray/io/json/json_grids_io.hpp"
+#include "detray/io/json/json_material_io.hpp"
```

Comment:
Hm, yes, I am not too happy about the naming myself, but just ```json``` is maybe a bit general. I may switch the names between ```json``` and ```json_def```

Reply:
```json_geometry_writer.hpp``` includes ```json_io.hpp``` for the definitions of ```to_json``` and ```from_json```, which are bundled in the ```json_io.hpp``` header. Won't this become a cyclic dependency if I include that there, too?

I renamed ```json_io.hpp``` to ```json_serializers.hpp``` and ```json_defs.hpp``` to ```json.hpp``` now, since it contains the nlohmann include